client/signup/steps/domains/index.jsx

```diff
@@ -617,10 +638,7 @@ class DomainsStep extends Component {
 		}
 
 		if ( isReskinned ) {
-			return (
-				! stepSectionName &&
-				translate( "Enter your site's name or some descriptive keywords to get started" )
-			);
+			return ! stepSectionName && translate( 'Enter some descriptive keywords to get started' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.